apsuite/commisslib/measure_orbit_stability.py

```diff
@@ -534,15 +551,64 @@ def _get_switching_freq(self):
 
     # static methods
     @staticmethod
-    def filter_matrix(matrix, fmin=0, fmax=None, fs=1):
-        """."""
-        if fmax is None:
-            fmax = fs/2
+    def filter_matrix(matrix, fmin, fmax, fsampling, keep_within_range=True):
```

Comment:
I don't like this name. Data does not need to be a matrix to be filtered. It filters each vector individually, so it is intrinsically a 1D algorithm. The only difference is that it is vectorized.

Reply:
Right. I renamed it to `filter_orbit_frequencies`